Source/WebCore/WebCoreSwift/CryptoKitShim.swift

```diff
@@ -23,22 +23,23 @@
 * THE POSSIBILITY OF SUCH DAMAGE.
 */
 
-import Foundation
```

Comment:
Most of the changes in the AESGCM implementation added in this PR:
https://github.com/WebKit/WebKit/pull/22731
are due to me running swift-format on the whole file.